addons/mail/static/tests/chatter_tests.js

```diff
@@ -221,6 +227,15 @@ QUnit.test('list activity widget with exception', async function (assert) {
 QUnit.test('list activity widget: open dropdown', async function (assert) {
     assert.expect(10);
 
+    const rpcList = [
+        '/web/dataset/search_read',
+        '/mail/init_messaging',
+        'switch_view',
+        'open dropdown',
+        'activity_format',
+        'action_feedback',
+        'read',
+    ];
```

Comment:
Would prefer `switch/case` in `mockRPC` like with the enterprise branch.
Also could turn all of them as route rather than having mix of routes and methods.

I don't mind listing the RPCs twice. I prefer that introducing this variable.